src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/index.css"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-sun calendar--month-31">
```

Comment:
The class name 'calendar--start-sun' uses a double hyphen, which is typically reserved for BEM modifiers. Ensure this is intentional and follows your project's naming conventions.